v8go.cc

```diff
@@ -134,16 +134,38 @@ IsolateHStatistics IsolationGetHeapStatistics(IsolatePtr ptr) {
 
 /********** Context **********/
 
+// Print provides a basic interface to print to stdout.
+static void Print(const FunctionCallbackInfo<Value>& args) {
+  bool first = true;
+  auto iso = args.GetIsolate();
+  for (int i = 0; i < args.Length(); i++) {
+    HandleScope handle_scope(iso);
+    if (first) {
+      first = false;
+    } else {
+      printf(" ");
+    }
+    String::Utf8Value value(iso, args[i]);
+    printf("%s", *value);
+  }
+  printf("\n");
+  fflush(stdout);
+}
+
 ContextPtr NewContext(IsolatePtr ptr) {
     Isolate* iso = static_cast<Isolate*>(ptr);
     Locker locker(iso);
     Isolate::Scope isolate_scope(iso);
     HandleScope handle_scope(iso);
   
     iso->SetCaptureStackTraceForUncaughtExceptions(true);
+
+    v8::Local<v8::ObjectTemplate> global = v8::ObjectTemplate::New(iso);
+    global->Set(v8::String::NewFromUtf8(iso, "print", NewStringType::kNormal).ToLocalChecked(),
```

Comment:
Would it not make more sense to use a familiar JavaScript API; for example `console.log` `console.error` etc

Reply:
Need to think about this API in a bit more depth. If we are looking to provide some sort of interop then maybe we need a `v8go` object.... eg: `v8go.print()`; That said I'm not 100% convinced in having "custom" code withing your Javascript code.